fc3/settings/base.py

```diff
@@ -78,7 +78,6 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.doc.XViewMiddleware',
```

Comment:
XViewMiddleware was removed in django 1.8.x. and is no longer supported. We may need to add something to replace it.